macros/GroupData.json

```diff
@@ -1822,10 +1822,33 @@
         },
         "WebRTC": {
             "overview":   [ "WebRTC API" ],
+            "guides":     [ { "url":   "/en-US/docs/Web/API/WebRTC_API/Protocols",
+                              "title": "Introduction to WebRTC protocols" },
+                            { "url":   "/en-US/docs/Web/API/WebRTC_API/Connectivity",
+                              "title": "WebRTC connectivity" },
+                            { "url":   "/en-US/docs/Web/API/WebRTC_API/Session_lifetime",
+                              "title": "Lifetime of a WebRTC session" },
+                            { "url":   "/en-US/docs/Web/API/WebRTC_API/Signaling_and_video_calling",
+                              "title": "Signaling and video calling" },
+                            { "url":   "/en-US/docs/Web/API/WebRTC_API/Using_data_channels",
+                              "title": "Using WebRTC data channels" },
+                            { "url":   "/en-US/docs/Web/API/WebRTC_API/Using_DTMF",
+                              "title": "Using DTMF with WebRTC" },
+                            { "url":   "/en-US/docs/Web/API/WebRTC_API/adapter.js",
+                              "title": "Improving compatibility using WebRTC adapter.js" },
+                            { "url":   "/en-US/docs/Web/API/WebRTC_API/Taking_still_photos",
+                              "title": "Taking still photos with WebRTC" },
+                            { "url":   "/en-US/docs/Web/API/WebRTC_API/Simple_RTCDataChannel_sample",
+                              "title": "A simple RTCDataChannel example" } ],
+
             "interfaces": [ "MediaStreamEvent",
                             "RTCConfiguration",
                             "RTCDataChannel",
                             "RTCDataChannelEvent",
+                            "RTCDTMFSender",
+                            "RTCDTMFToneChangeEvent",
+                            "RTCIceCandidate",
+                            "RTCIceTransport",
```

Comment:
RTCDTMFSender: https://developer.mozilla.org/en-US/docs/Web/API/RTCDTMFSender (https://dxr.mozilla.org/mozilla-central/source/dom/webidl/RTCDTMFSender.webidl)

RTCDTMFToneChangeEvent: https://developer.mozilla.org/en-US/docs/Web/API/RTCDTMFToneChangeEvent (https://dxr.mozilla.org/mozilla-central/source/dom/webidl/RTCDTMFToneChangeEvent.webidl)

RTCIceCandidate: https://developer.mozilla.org/en-US/docs/Web/API/RTCIceCandidate (https://dxr.mozilla.org/mozilla-central/source/dom/webidl/RTCIceCandidate.webidl)

RTCIceTransport: Not implemented yet although its functionality is somewhat duplicated. There's a bug filed to implement it: https://bugzilla.mozilla.org/show_bug.cgi?id=1307994